app/models/booking.rb

```diff
@@ -2,9 +2,8 @@ class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :chalet
 
-  validates :user_id, presence: true
-  validates :chalet_id, presence: true
-  validates :check_in_date, presence: true
-  validates :check_out_date, presence: true
+  validates :user_id, :chalet_id, :check_in_date, :check_out_date, presence: true
+  validates :status, presence: true, inclusion: { in: ["Approved", "Declined", "Pending"] }
```

Comment:
Looks good. 
Just to double check - can you combine presence & inclusion on one line of code?

Reply:
Yep, you can! :) @alimlalji1